core/templates/dev/head/base.js

```diff
@@ -25,10 +25,13 @@ oppia.constant('PLACEHOLDER_OUTCOME_DEST', '/');
 
 oppia.constant('DEFAULT_RULE_NAME', 'Default');
 
-oppia.constant('FUZZY_RULE_TYPE', 'FuzzyMatches');
+oppia.constant('CLASSIFIER_RULE_TYPE', 'ClassifyMatches');
 
-oppia.constant('DEFAULT_FUZZY_RULE', {
-  rule_type: 'FuzzyMatches',
+// Feature still in development. Synchronize this with feconf.
```

Comment:
Reads a bit like a TODO. Maybe:

    Feature still in development.
    NOTE TO DEVELOPERS: This should be synchronized with the value in feconf.